docs/source/cli/model-training-inference/distributed/sagemaker.rst

```diff
@@ -213,6 +213,105 @@ from ``${DATASET_S3_PATH}`` as input and create a DistDGL graph with
 ``${NUM_PARTITIONS}`` under the output path, ``${OUTPUT_PATH}``.
 Currently we only support ``random`` as the partitioning algorithm.
 
+Launch hyper-parameter optimization task
```

Comment:
Why is the doc here in this PR? I suppose it should be in HPO tuning related PR.